lib/s3-upload.ts

```diff
@@ -2,46 +2,43 @@ import { ReadStream } from "fs";
 import fs from "fs";
 import { v4 as uuid } from "uuid";
 import { UploadResult } from "./types";
-import { S3Client, PutObjectCommand, GetObjectCommand } from "@aws-sdk/client-s3";
-import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { logMessage } from "./logger";
+import { S3Client, PutObjectCommand, DeleteObjectCommand } from "@aws-sdk/client-s3";
+
 import formidable from "formidable";
 
-const s3Client = new S3Client({ region: "ca-central-1" });
+const s3Client = new S3Client({ region: process.env.AWS_REGION ?? "ca-central-1" });
+
+const bucketName: string =
+  process.env.RELIABILITY_FILE_STORAGE ?? "forms-staging-reliability-file-storage";
 
 /**
  * This function tries to upload a given file to aws S3 bucket and returns a data object
  * which stores an url.
  * @param file
- * @param bucketName
- * @param filePath
  * @returns
  */
-const uploadFileToS3 = async (
-  file: formidable.File,
-  bucketName: string,
-  filePath: string
-): Promise<UploadResult> => {
+const uploadFileToS3 = async (file: formidable.File): Promise<UploadResult> => {
```

Comment:
refactored to use only the file as a parameter.  `filePath` parameter with the file name which is already included on the `file` object.